### PR TITLE
add additional config for oaiprovider from @jefferya

### PR DESCRIPTION
**Pull request by pgwillia**
part of https://github.com/ualbertalib/jupiter/issues/472

### CODE REVIEW

**Review by murny — APPROVED** [review]

File: oaiprovider-4.7/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+FROM tomcat:8.5
+
+ENV FCREPO_VERSION 4.7.4.3
+
+# add authentication credentials to Tomcat for Fcrepo
+# remove the closing tag, then use the echo|tee pattern to build it back in
+RUN sed -i 's|</tomcat-users>| |' $CATALINA_HOME/conf/tomcat-users.xml
+RUN echo "<role rolename=\"fedoraAdmin\" />" | tee -a $CATALINA_HOME/conf/tomcat-users.xml \
+    && echo "<user username=\"${FCREPO_ADMIN_USER:-fedoraAdmin}\" password=\"${FCREPO_ADMIN_PASSWORD:-fedoraAdmin}\" roles=\"fedoraAdmin\"/>" | tee -a $CATALINA_HOME/conf/tomcat-users.xml \
```
Comment (pgwillia) [inline oaiprovider-4.7/Dockerfile:9]: This approach doesn't look like it works to set password during `docker run` back to the drawing board :(

Comment (murny) [reply oaiprovider-4.7/Dockerfile:9]: Should be able to something like:

`ENV FCREPO_ADMIN_USER fedoraAdmin`

Then in this command reference that ENV via `$FCREPO_ADMIN_USER`

Then you can set this ENV to override the default from the `docker run` or `docker-compose.yml`

Comment (pgwillia) [reply oaiprovider-4.7/Dockerfile:9]: That's what I thought I was doing but it turns out RUN isn't supported https://docs.docker.com/engine/reference/builder/#environment-replacement

Comment (murny) [reply oaiprovider-4.7/Dockerfile:9]: I don't see anything about ENV vars not being supported in your link?

See also here: https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e-env-env-file

Comment (pgwillia) [reply oaiprovider-4.7/Dockerfile:9]: Environment variables are supported by the following list of instructions in the Dockerfile:

    ADD
    COPY
    ENV
    EXPOSE
    FROM
    LABEL
    STOPSIGNAL
    USER
    VOLUME
    WORKDIR

as well as:

    ONBUILD (when combined with one of the supported instructions above)

Comment (murny) [reply oaiprovider-4.7/Dockerfile:9]: Yes, I was suggesting to use the 3rd one on that list:

`ENV`

So you can set an `ENV` for FCREPO_ADMIN_USER and FCREPO_ADMIN_PASSWORD in the dockerfile.

(much like how FCREPO_VERSION is being set in the dockerfile, so theoretically you could change the version just by overriding this env var without having to create a new dockerfile)

Then be able to override it from outside the dockerfile, when using docker-compose or docker run

**Pull request merged**